source/postprocess/heating_statistics.cc

```diff
@@ -118,10 +118,7 @@ namespace aspect
                            global_heating_integrals);
       global_mass = Utilities::MPI::sum (local_mass, this->get_mpi_communicator());
 
-      unsigned int index = 0;
-      for (typename std::list<std::unique_ptr<HeatingModel::Interface<dim>>>::const_iterator
-           heating_model = heating_model_objects.begin();
-           heating_model != heating_model_objects.end(); ++heating_model, ++index)
+      for (unsigned int index=0; index<headting_model_objects.size(); ++index)
```

Comment:
Little typo, please fix. This is why the testers fail:
```suggestion
      for (unsigned int index=0; index<heating_model_objects.size(); ++index)
```

Reply:
Hopefully this is fixed now @gassmoeller